processor/transform/tables-to-jsx.ts

```diff
@@ -19,6 +19,8 @@ const alignToStyle = (align: 'left' | 'center' | 'right' | null) => {
 
 const isTableCell = (node: Node) => ['tableHead', 'tableCell'].includes(node.type);
 
+const isLiteral = (node: Node): node is Literal => 'value' in node;
```

Comment:
Wait, I know about `as` but what is the meaning of `is` in TS???

Reply:
I don't know the term off hand, but it means the function is restricting the type?